module/VuFindSearch/src/VuFindSearch/Backend/WorldCat2/Backend.php

```diff
@@ -115,7 +115,7 @@ public function search(
      */
     public function retrieve($id, ?ParamBag $params = null)
     {
-        $response   = $this->connector->getRecord($id, $params);
+        $response   = $this->connector->getRecord($id);
```

Comment:
This refactoring is correct, but it now requires that we add a `@SuppressWarnings(PHPMD.UnusedFormalParameter)` annotation to the function header above since it requires us to ignore one of the standard interface parameters.